src/chunk.c

```diff
@@ -1691,11 +1692,31 @@ chunk_tuple_found(TupleInfo *ti, void *arg)
 	 * ts_chunk_build_from_tuple_and_stub() since chunk_resurrect() also uses
 	 * that function and, in that case, the chunk object is needed to create
 	 * the data table and related objects. */
-	chunk->table_id = get_relname_relid(chunk->fd.table_name.data,
-										get_namespace_oid(chunk->fd.schema_name.data, true));
+	Oid namespace_oid = get_namespace_oid(chunk->fd.schema_name.data, true);
+
+	Ensure(OidIsValid(namespace_oid),
+		   "namespace for schema %s has invalid OID",
+		   quote_identifier(NameStr(chunk->fd.schema_name)));
```

Comment:
```suggestion
	Oid namespace_oid = get_namespace_oid(chunk->fd.schema_name.data, false);
```

Reply:
The 2nd argument of get_namespace_oid controls whether the namespace is required to exist so we should make use of that instead of writing our own check.